RELEASE.md

```diff
@@ -18,6 +18,8 @@ It should look like:
 (snip)
 ## [Unreleased]
 
+## [0.5.3]
```

Comment:
What's this?

Reply:
Before the change, It seems like the description was added to `Unreleased`.

Reply:
I can't understand.  Could you explain in other words?

Reply:
Revert it, because there was no need to change.